conf/application.conf

```diff
@@ -18,7 +18,7 @@ appName = "fh-registration-frontend"
 play.http.router = prod.Routes
 
 play {
-  filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 localhost:9250 stats.g.doubleclick.net http://localhost:12345 www.googletagmanager.com www.google-analytics.com data:"
+  filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 localhost:9250 stats.g.doubleclick.net http://localhost:12345 www.googletagmanager.com data:"
```

Comment:
We have just learnt we should leave the CSP alone, goose-analytics should not be removed from it, so better left as is

Reply:
It's reverted back